reference/glossary.md

```diff
@@ -0,0 +1,445 @@
+
+See [Bootstrap process](https://discourse.ubuntu.com/t/about-applications/17760#bootstrap).
+
+<a name="container"></a>
+### Container
+
+One of the main objects of Anbox Cloud. Every time you launch an application or image, Anbox Cloud creates a container for it. Every container provides a full Android system.
+
+See https://discourse.ubuntu.com/t/about-containers/17763.
+
+<a name="core-stack"></a>
+### Core stack
+
+The core parts of the Anbox Cloud stack that are required for all deployments.
+
+See https://discourse.ubuntu.com/t/about-anbox-cloud/17802.
+
+<a name="coturn"></a>
+### Coturn
+
+An open-source implementation of a STUN/TURN server needed for WebRTC to work behind NATs and firewalls.
+
+See [the Coturn project on GitHub](https://github.com/coturn/coturn).
+
+<a name="grafana"></a>
+### Grafana
+
+A tool for analytics and monitoring that allows to query and visualise the metrics of the cluster or individual containers and generate alerts based on the data.
+
+See https://discourse.ubuntu.com/t/use-grafana/17787 and [the Grafana website](https://grafana.com/).
+
+<a name="gpu"></a>
+### Graphics Processing Unit (GPU)
+
+A specialised processor that is designed to accelerate image processing and graphics rendering for output to a display device.
+
+<a name="ha"></a>
+### High availability (HA)
+
+The characteristic of a system to continuously be available without failing for a higher-than-normal period of time.
+
+See https://discourse.ubuntu.com/t/enable-high-availability/17754.
+
+<a name="hook"></a>
+### Hook
+
+Code that is invoked at different points in time in the life cycle of a container. Hooks are part of addons.
+
+See [Hooks](https://discourse.ubuntu.com/t/addons/25293#hooks).
+
+<a name="image"></a>
+### Image
+
+The base for a container, which contains all necessary components like Anbox or the Android root file system. Anbox Cloud provides images based on different Android versions and different architectures.
+
+See https://discourse.ubuntu.com/t/manage-images/17758 and https://discourse.ubuntu.com/t/provided-images/24185.
+
+<a name="instance-type"></a>
+### Instance type
+
+An abstraction for a set of resources that is available to a container.
+
+See https://discourse.ubuntu.com/t/instance-types/17764.
+
+<a name="juju"></a>
+### Juju
+
+A charmed operator framework that helps you deploy, integrate and manage applications across multiple environments. Anbox Cloud is installed using Juju. The Anbox Cloud Appliance uses Juju under the hood.
+
+See [the Juju website](https://juju.is/).
+
+<a name="lxd"></a>
+### LXD
+
+A system container and virtual machine manager that offers a unified user experience around full Linux systems running inside containers or virtual machines. Anbox Cloud is based on LXD.
+
+See [the LXD website](https://linuxcontainers.org/).
+
+<a name="monitoring-stack"></a>
+### Monitoring stack
+
+The part of the Anbox Cloud stack that provides monitoring functionality.
```

Comment:
If I understood it correctly, the monitoring stack is an optional part of the core stack? (Which contradicts the definition of the core stack ;) ).

Reply:
It's a recommended but indeed optional part of the stack. You can still run containers without it

Reply:
I would like to rephrase this here and make the "monitoring stack" a generic thing which can be implemented in different ways. We as Anbox Cloud team don't provide such a thing but can make recommendations what to use.